### PR TITLE
check permissions before rollback

### PR DESCRIPTION
**Pull request by schiessle**
@LukasReschke please have a look, thanks!

### CODE REVIEW

File: apps/files_versions/lib/Storage.php [hunk 1]
```diff
@@ -324,6 +324,14 @@ public static function rollback($file, $revision) {
 			$files_view = new View('/'. User::getUser().'/files');
 			$versionCreated = false;
 
+			$fileInfo = $files_view->getFileInfo($file);
+			$permission = $fileInfo->getPermissions();
```
Comment (LukasReschke) [inline apps/files_versions/lib/Storage.php]: Unused?

**Comment by LukasReschke** [comment]
Works like a charm 🚀 👍

**Comment by MorrisJobke** [comment]
Works 👍

**Comment by MorrisJobke** [comment]
@schiessle Please backport

**Pull request merged**